now.json

```diff
@@ -2,15 +2,9 @@
   "name": "primer-components",
   "version": 2,
   "alias": "primer-components.now.sh",
-  "files": [
-    ".babelrc",
-    "package-lock.json",
-    "docs",
-    "rollup.config.js",
-    "src",
-    "static"
-  ],
+  "files": [".babelrc", "package-lock.json", "docs", "rollup.config.js", "src", "static"],
   "routes": [
+    {"src": "/playroom(/.*)?", "dest": "$1"},
```

Comment:
```suggestion
    {"src": "/components(/.*)?", "dest": "/playground$1"},
```

Reply:
This doesn't  work 😢  `primer.style/components/playroom` was my original plan but I've tried all sorts of things and can't get it to work. I agree that `/components/playroom` makes more sense but it might not be worth the hassle right now. Would you be okay with just `/playroom` for now, until we can get `/components/playroom` working?

Reply:
Should this be `/components/playroom`? `/components` is already mapped on the line below

Reply:
^ I tried that. No dice

Reply:
I figured `{"src": "/components/playroom(/.*)?", "dest": "$1"}` was our best bet. However, with that setup, the page 404s when loading all the JavaScript files because it's sending requests to `localhost:9000/components/runtime.js` instead of `localhost:9000/runtime.js`.

Reply:
I'm fine with using `/playroom` for now!

Reply:
It looks like line 8 404's anyways, not sure if it's even being used anymore